data-products/src/new_tab_recommendations/aggregate_engagement_flow.py

```diff
@@ -189,16 +222,27 @@ async def export_telemetry_by_corpus_item_id(
 
 
 @flow(name="new-tab-recommendations.aggregate-engagement")
-async def aggregate_engagement():
+async def aggregate_engagement(region: str = None):
     """
     Ingests NewTab telemetry joined with Corpus metadata into a Sagemaker Feature Group.
     """
     logger = get_run_logger()
+
+    # Do an incremental update of the events table.
+    await bigquery_query(
+        gcp_credentials=MozGcp(), query=UPDATE_POCKET_USER_EVENTS_BY_COUNTRY_SQL
+    )
```

Comment:
This query should run before the ones below. Is it still necessary to define a dependency on those tasks in Prefect v2?

If so, would the cleanest way to do that be to add `if result is not None:`?